java/src/main/resources/config.properties

```diff
@@ -1,5 +1,8 @@
 web.server.port=8080
 
+crate.user=crate
+crate.password=crate
```

Comment:
since the `crate` superuser does not have any password I would leave this property empty.